man/systemd-measure.xml

```diff
@@ -104,6 +104,7 @@
         <term><option>--initrd=PATH</option></term>
         <term><option>--splash=PATH</option></term>
         <term><option>--dtb=PATH</option></term>
+        <term><option>--pcrpkey=PATH</option></term>
```

Comment:
this is a bit of a mouthful - as a PE section name it might make sense to keep it short, but as a command line parameter can it be expanded to something readable? having both is also fine

Reply:
It's really incomprehensible as a command line switch - can you at least add a readable one as an alias? So both can stay and be documented

Reply:
It sure is, but I still think its's easier to stick to the literal value here, since it directly maps to the objcopy command line that "system-measure sign" is usually paired with. i.e. from the example in the manpage:

```
 systemd-measure sign \
     --linux=vmlinux \
     --osrel=os-release.txt \
     --cmdline=cmdline.txt \
     --initrd=initrd.cpio \
     --splash=splash.bmp \
     --dtb=devicetree.dtb \
     --pcrpkey=tpm2-pcr-public.pem \
     --bank=sha1 \
     --bank=sha256 \
     --private-key=tpm2-pcr-private.pem \
     --public-key=tpm2-pcr-public.pem > tpm2-pcr-signature.json
```

and 

```
# objcopy \
    --add-section .linux=vmlinux --change-section-vma .linux=0x2000000 \
    --add-section .osrel=os-release.txt --change-section-vma .osrel=0x20000 \
    --add-section .cmdline=cmdline.txt --change-section-vma .cmdline=0x30000 \
    --add-section .initrd=initrd.cpio --change-section-vma .initrd=0x3000000 \
    --add-section .splash=splash.bmp --change-section-vma .splash=0x100000 \
    --add-section .dtb=devicetree.dtb --change-section-vma .dtb=0x40000 \
    --add-section .pcrsig=tpm2-pcr-signature.json --change-section-vma .splash=0x80000 \
    --add-section .pcrpkey=tpm2-pcr-public.pem --change-section-vma .splash=0x90000 \
    /usr/lib/systemd/boot/efi/linuxx64.efi.stub \
    foo.efi
```

i.e. the first 7 lines or so map to each other 1:1. it's very systematic in that. I think it would be a lot more confusing if we'd have to specify the sections under one name in systemd-measure and another one in objcopy. Simplest is to really reause the literal same spelling for both.

Reply:
why not have an alias? then that still works, but the first impact on --help is way less confusing